src/darwin/Framework/chip_xcode_build_connector.sh

```diff
@@ -61,29 +61,14 @@ done
 target_defines=[${target_defines:1}]
 
 declare target_cpu=
-case $PLATFORM_PREFERRED_ARCH in
-    i386)
-        target_cpu=x86
-        ;;
-    x86_64)
-        target_cpu=x64
-        ;;
-    armv7)
-        target_cpu=arm
-        ;;
-    arm64)
-        target_cpu=arm64
-        ;;
-    *)
-        echo >&2
-        ;;
-esac
-
 declare target_cflags=""
 
 read -r -a archs <<<"$ARCHS"
 
 for arch in "${archs[@]}"; do
+    if [ -z "$target_cpu"] ]; then
+        target_cpu="$arch"
```

Comment:
The case statement being removed above had different values for PLATFORM_PREFERRED_ARCH and `target_cpu`.  Are the values in ARCHS not the same as the values that can end up in PLATFORM_PREFERRED_ARCH?

Reply:
Yes, I probably need to handle `x86_64` -> `x64`. I don't think `armv7` and `i386` are actually relevant for Darwin builds?

Reply:
Probably not, yeah.